app/controllers/concerns/file_upload_limitable.rb

```diff
@@ -0,0 +1,16 @@
+module FileUploadLimitable
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :single_file_upload_limit, only: [:create]
```

Comment:
can we rename this something like `allow_only_one_attachment`

Reply:
Yes that makes more sense. Any other naming suggestions regarding the concern? Nothing sounded great when I made it but it "works"

Reply:
The way we have this now using a before action within the concern will limit the expandability of this concern. We could go a bit more generic on the naming and remove the before action to the controller as needed. So in this case we could call the concern something like AttachmentManageable and then we can keep a wider variety of attachment related methods in here. Does that resonate?